### ISSUE TITLE
Fails with 5.38

### ISSUE THREAD

**Issue by andk** [dgl/re-engine-RE2#15]
Sample fail report: http://www.cpantesters.org/cpan/report/ee062c2a-0d9e-11ee-bf69-46044ed2e7f5

In the commit

``` text
commit c0bd4734ae7c7bb5045ad389161fac511cc8544d
Author: Yves Orton <demerphq@gmail.com>
Date:   Fri Jan 27 16:25:48 2023 +0800

    fixup re::engine::RE2 for perl 5.37.8 and later (#10)
```

I found this piece:

``` text
+#if PERL_VERSION >= 37 && PERL_SUBVERSION >=7
+    rx->logical_nparens = rx->nparens;
+#endif
```

@demerphq , you may be interested, this does not cover 5.38, right?

**Comment by demerphq** [comment]
Stupid math error. Grrr. It needs to be fixed to handle higher version
numbers.

Yves

**Comment by dgl** [comment]
Fixed in 171cc4f.
